helm/tests/ensure.py

```diff
@@ -742,6 +742,15 @@ def kubeadm_control_plane(kubernetes_cluster):
         spec:
           kubeadmConfigSpec:
             clusterConfiguration:
+              apiServer:
+                extraArgs:
+                  cloud-config: /etc/kubernetes/azure.json
```

Comment:
I'm adding these to the common fixture so we can later assert that we are not overwriting existing configuration.